tests/unittests/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) Microsoft. All rights reserved
+#
+
+include $(CLEAR_VARS)
+
+CLANG := clang++
+
+SRCDIR := Foundation
+SRCFILES := $(wildcard $(SRCDIR)/*.m*)
+RF_SRCFILES := $(wildcard $(SRCDIR)/ReferenceFoundation/*.mm)
```

Comment:
These two could be combined, since it doesn't look like you're doing anything separate with them. Or,
```
SRCDIRS := Foundation Foundation/ReferenceFoundation
SRCFILES := $(foreach d,$(SRCDIRS),$(wildcard $(d)/*.m*))
```